articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+---
+title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+description: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Site Recovery
+toc_sub3:
+toc_sub4:
+toc_title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+toc_fullpath: Operators/How To/azs-how-setup-recovery-portal.md
```

Comment:
Users (not Operators)

Add Site Recovery to the path

Reply:
done